tests/network/localnet/migration_stuntime/libstuntime.py

```
@@ -21,6 +22,20 @@ class InsufficientStuntimeDataError(ValueError):
     """Raised when ping log has too few successful replies to compute stuntime."""
 
 
+def measure_stuntime(active_ping: "ContinuousPing") -> float:
```

Ruff (0.15.12)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)